astrokat/__init__.py

```diff
@@ -1,11 +1,12 @@
+"""."""
```

Comment:
Not sure if this was just a way to get flake8 to pass or not. But a docstring should describe your source code - i.e. what your function, module, or class does. It shouldn't just be a period.